themes/openideal_theme/openideal_theme.theme

```diff
@@ -185,3 +185,18 @@ function openideal_theme_preprocess_image(&$variables) {
     $variables['attributes']['title'] = $variables['alt'];
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function openideal_theme_preprocess_message__user_mention(&$variables) {
+  // In case the user is not viewing its page,
+  // its necessary to change the partial template.
+  if (\Drupal::routeMatch()->getRouteName() == 'entity.user.canonical' &&
+    \Drupal::currentUser()->id() !== \Drupal::routeMatch()->getParameter('user')->id()) {
+    $elements = &$variables['elements'];
+    $variables['#cache']['contexts'][] = 'user.is_user_profile_page';
+    $partials = $elements['#message']->getText(\Drupal::languageManager()->getCurrentLanguage()->getId());
```

Comment:
`getText()` accepts one more argument - delta. Can we pass "3" to get correct text line?